backend/tests/local/test-schedules.sh

```
@@ -0,0 +1,130 @@
+            fired=1
+            break
+        fi
+    done
+    [ $fired -eq 0 ] && print_fail "  No fires observed in 75s — pg_cron may not be running"
+else
+    print_info "5) Skipping fire-poll (set SCHEDULES_WAIT_FOR_FIRE=1 to enable)"
+fi
+
+cleanup_schedules "$ADMIN_TOKEN"
+
+if [ $TEST_FAILED -eq 1 ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)